Makefile

```diff
@@ -45,3 +45,6 @@ clean:
 # @TODO consider a DX-friendly way to ensure dynamic .env file is supported
 build-frontend:
 	cd frontend && yarn && yarn build
+
+dev-frontend:
+	cd frontend && yarn develop
```

Comment:
During development, you should also concurrently run `yarn generate:css`

Reply:
Just curious: i ran `yarn develop` on my local, and that was all I need to access a working local deployment. What's the value of doing `yarn generate:css` here? Perhaps, it watches for the changes in CSS files & rebuilds / reloads automatically for us? 🙂